### PR TITLE
Add Login and Sign Up with JWT Token

### PR DESCRIPTION
**Pull request by ThomasF34**

### CODE REVIEW

File: src/routes/login.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  })(req, res, next);
+
+  // const parent = new Parent({
+  //   email: 'toto',
+  //   firstname: 'Thomas',
+  //   lastname: 'Falcone',
+  // });
+  // await parent.setPassword('tamere');
+  // parent.save();
+  // res.send('C"est fait');
```
Comment (MathieuVeber) [inline src/routes/login.ts]:
```suggestion
```

It should be removed from the file

File: src/routes/register.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    // Assigning a token
+    const token = jwt.sign({
+      _id: newParent.id
+    }, process.env.TOKEN_SECRET || 'secret');
+
+    res.header('Set-Cookie', token)
+      .status(200)
+      .json({
+        parent: newParent.getSafeParent()
+      });
```
Comment (MathieuVeber) [inline src/routes/register.ts]: It should be indented correctly

**Review by MathieuVeber — CHANGES_REQUESTED** [review]
Great job overall ! Some changes though

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# La Cabane Aux Etoiles
+
+## Install and run locally
+```
+npm install
+npm run dev
```
Comment (MathieuVeber) [inline README.md:6]: Are you sure this is the correct command ?

Comment (ThomasF34) [reply README.md:6]: Yes totally 😄

**Review by MathieuVeber — APPROVED** [review]
LGTM 👍

**Comment by MathieuVeber** [comment]
[Simple Login](https://trello.com/c/38cJnWte/33-simple-login)

**Comment by MathieuVeber** [comment]
[Simple Register](https://trello.com/c/yZTHR0hk/35-simple-register)

**Pull request merged**